packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -390,7 +390,9 @@ export interface TreeArrayNodeUnsafe<TAllowedTypes extends Unenforced<ImplicitAl
 
 // @public @sealed
 export interface TreeChangeEvents {
-    nodeChanged(): void;
+    nodeChanged({ changedProperties }: {
+        changedProperties: ReadonlySet<string>;
```

Comment:
I'm a little concerned about forward compat (for when we finally expose information about what the changes were).
Is there a design for what that will look like?
Are we expecting to just deprecate `changedProperties` and use another key for the future API? (`propertyChanges` ?)
Or would we be better off exposing `changedProperties: ReadonlyMap<string, unknown>` for now, so we can fill in the blanks later?

Reply:
I think the fact that `changedProperties` is inside of a property bag `{...}` (as opposed to directly being the first parameter to the event) is sufficient for future-proofing and about as good of a guess as we can make. To add more information in the future, we don't need to break the event signature, we just add more properties to the event bag. Maybe there's some overlap with the existing properties and a new property supersedes an old one, but that's fine, the user can just ignore the deprecated one.

Reply:
I'm cool with that so long as we're going into it with our eyes open.
Feel free to resolve this comment without further changes.